src/components/tiptapEditor/calloutExtension.jsx

```diff
@@ -0,0 +1,85 @@
+import { Node, mergeAttributes } from '@tiptap/core';
+
+const CalloutExtension = Node.create({
+  name: 'callout',
+
+  group: 'block',
+
+  content: 'block+',
+
+  addAttributes() {
+    return {
+      type: {
+        parseHTML: element => element.getAttribute('type'),
+        renderHTML: attributes => ({ 'type': attributes.type }),
+      },
+    };
+  },
+
+  parseHTML() {
+    return [
+      {
+        tag: 'div[data-callout]',
+      },
+    ];
+  },
+
+  renderHTML({ HTMLAttributes }) {
+    const { type } = HTMLAttributes;
+    const icons = {
+      info: "ℹ️",
+      warning: "⚠️",
+      error: "❌",
+      success: "✅",
+      tip: "💡",
+      important: "📌",
+    };
+
+    return [
+      "div",
+      mergeAttributes(
+        {
+          "data-callout": "",
+          type: type,
+          "data-id": HTMLAttributes.id || Math.random().toString(36).substr(2, 9), // Unique ID
+          class: `callout callout-${type}`,
+        },
+        HTMLAttributes
+      ),
+      [
+        "div",
+        { class: "callout-container", style: "position: relative;" },
```

Comment:
use bootstrap class position-relative
```suggestion
        { class: "callout-container position-relative" },
```